subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -1980,6 +2008,12 @@ SHELL_STATIC_SUBCMD_SET_CREATE(wifi_commands,
 		     cmd_wifi_ps_wakeup_mode,
 		     2,
 		     0),
+	SHELL_CMD_ARG(rts_threshold,
+		     NULL,
+		     "<rts_threshold: (0-6500631>.\n",
```

Comment:
```suggestion
		     "<rts_threshold: (0-6500631)>.\n",
```

Do we need to put the range here? At least use the macro to avoid updating in two places.